dashboards_bundlers/server_upload/__init__.py

```diff
@@ -92,14 +92,12 @@ def send_file(file_path, dashboard_name, handler):
             if result.status_code >= 400:
                 raise web.HTTPError(result.status_code)
 
-        # Redirect for client might be different from internal upload URL
-        redirect_server = os.getenv('DASHBOARD_REDIRECT_URL')
-        if redirect_server:
-            redirect_root = redirect_server.format(hostname=hostname,
-                port=port, protocol=protocol)
-        else:
-            redirect_root = dashboard_server
-        handler.redirect(url_path_join(redirect_root, VIEW_ENDPOINT, escape.url_escape(dashboard_name, False)))
+        # Redirect to link specified in response body
+        res_body = result.json()
+        redirect_link = res_body['link']
+        if not redirect_link:
+            redirect_link = url_path_join(dashboard_server, VIEW_ENDPOINT, escape.url_escape(dashboard_name, False))
```

Comment:
Should we fall back on the `DASHBOARD_REDIRECT_URL` option and use it if the dashboard server does not return a redirect link? Otherwise, certain apps will be broken until 0.6 of dashboard server is out.